### PR TITLE
Upgrade kui from 10.4.6 to 10.4.13

### PR DESCRIPTION
**Pull request by starpit**
Note: This PR also **adds an arm64 download option**, and a zap stanza.

---

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/kui.rb [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
   name "Kui"
   desc "Hybrid command-line/UI development experience for cloud-native development"
   homepage "https://kui.tools/"
 
   livecheck do
     url :url
-    strategy :github_latest
+    strategy :git
+    regex(/^v(\d+(?:\.\d+)*)$/)
   end
```
Comment (bevanjkay) [inline Casks/kui.rb:24]: @starpit I think the `:github_latest` strategy is probably still required as some tags do not include binaries. Also, for reference, when the strategy is `:git` and a custom regex is not required (and the download URL is from Github), the livecheck block should be removed as it is used by default.

**Review by bevanjkay — CHANGES_REQUESTED** [review]

**Comment by starpit** [comment]
Awesome thanks for the great feedback!

File: Casks/kui.rb (hunk 1 above)
Comment (starpit) [reply Casks/kui.rb]: Awesome thanks for the great feedback! I have pushed some updates.

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thanks @starpit

**Pull request merged**